go/test/endtoend/vtorc/primaryfailure/main_test.go

```diff
@@ -32,7 +32,7 @@ func TestMain(m *testing.M) {
 	var cellInfos []*utils.CellInfo
 	cellInfos = append(cellInfos, &utils.CellInfo{
 		CellName:    utils.Cell1,
-		NumReplicas: 12,
+		NumReplicas: 13,
```

Comment:
Why 13?

Reply:
The way the tests run is that they don't bring up a MySQL primary once it has crashed, so we bring up a certain number of tablets in a pool and each test requests for the subset of them that they need. Since I have added a new test in which I permanently remove a tablet from the pool, I had to increase its count by one.